### PR TITLE
Undelete split

### PR DESCRIPTION
**Pull request by swansontec**
This is completely untested code! I just want to push this up to show you what I was thinking.

### CODE REVIEW

File: src/modules/account/accountState.js [hunk 1]
```diff
@@ -483,6 +483,20 @@ class AccountState {
...
   }
 
+  listSplittableWalletTypes (walletId) {
+    const walletInfo = this.allKeys.find(
+      walletInfo => walletInfo.id === walletId
+    )
+    if (!walletInfo) throw new Error(`Invalid wallet id ${walletInfo.id}`)
+
+    // This is the bare minimum for now:
+    if (walletInfo.type === 'wallet:bitcoin') {
```
Comment (swansontec) [inline src/modules/account/accountState.js]: Probably need something like ` && walletInfo.keys.type !== 'bip49'`

Comment (thehobbit85) [reply src/modules/account/accountState.js]: Do you have access to the pluginCurrency info here?

Comment (swansontec) [reply src/modules/account/accountState.js]: I believe we can get access to the currencyInfo from there, yes. It should be `this.ai.props.state.currency.infos`. I think there is also a selector, `getCurrencyInfo`, so you could do `getCurrencyInfo(this.ai.props.state, walletType)`

Comment (thehobbit85) [reply src/modules/account/accountState.js]: If you access to the currencyInfos then we can do something like this:

```javascript
return pluginCurrencyInfoToSplitFrom.forks.filter(walletType => {
   pluginCurrencyInfoToSplitTo.formats.find(walletInfo.keys.format)
})
```

And I'll add 'formats' and 'forks' into the currencyInfos

**Pull request closed without merging**